### PR TITLE
feat(client): migrate layout/default to ts

### PR DESCRIPTION
**Pull request by aghoneim92**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Review by ShaunSHamilton — DISMISSED** [review]
Thank you, for taking the time to contribute, and for your patience as we got to this PR.

I have just resolved the conflicts, and fixed one type.

This LGTM 👍

File: client/src/declarations.d.ts [hunk 1]
```diff
@@ -9,6 +9,26 @@ declare module '*.svg' {
...
+            inlineMath: Array<string[]>;
+            processEscapes: boolean;
+            processClass: string;
+          };
+        }) => void;
+        Queue: (attributes: unknown[]) => void;
+      };
+    };
+  }
+}
```
Comment (nhcarrigan) [inline client/src/declarations.d.ts:31]: Is this not redundant with the `MathJax` declaration on line 44?

Comment (ShaunSHamilton) [reply client/src/declarations.d.ts:31]: I thought so, but last time I removed it, Linter complained.

**Review by ShaunSHamilton — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
I think I'm happy with this.

**Comment by nhcarrigan** [comment]
Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**